Meshtastic/Views/Onboarding/OnboardingView.swift

```diff
@@ -0,0 +1,315 @@
+import CoreBluetooth
+import OSLog
+import SwiftUI
+import Foundation
+import MapKit
+
+struct OnboardingView: View {
+	enum SetupGuide: Hashable {
+		case notifications
+		case location
+		case mqtt
+	}
+
+	@State var navigationPath: [SetupGuide] = []
+	@EnvironmentObject var bleManager: BLEManager
+
+	@Environment(\.dismiss) var dismiss
+
+	/// The Title View
+	var title: some View {
+		VStack {
+			Text("Welcome to")
+				.font(.title2.bold())
+				.multilineTextAlignment(.center)
+				.fixedSize(horizontal: false, vertical: true)
+			Text("Meshtastic")
```

Comment:
When strings don't need to be translated, you can use [`String(verbatim:)`](https://developer.apple.com/documentation/swiftui/text/init(verbatim:)):

```suggestion
			Text(verbatim: "Meshtastic")
```